lib/util/reduce_market_data.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+const { PAPER_MODE_PAIRS } = require("../constants");
+
+const INITIAL_MARKETS_OBJECT = {
+  sandboxMarkets: [],
+  liveMarkets: [],
+};
+
+module.exports = (marketData) => {
+  const marketValues = [...marketData.values()];
+
+  return marketValues.reduce((acc, m) => {
```

Comment:
based on the number of entries on marketData, I take back what I said about using reduce because this will create way too many array reallocations, please refactor using push and fixed-size arrays.